charts/m4d/templates/manager.yaml

```diff
@@ -405,13 +405,19 @@ spec:
       - args:
         - --enable-leader-election
         - --enable-all-controllers
```

Comment:
I think that this overrides the other flags.

Reply:
ok, will move them to values.yaml